qabelbox/src/main/res/menu/box_folder_bottom_sheet.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:id="@+id/open" android:title="@string/Open" android:icon="@drawable/ic_share_black"/>
```

Comment:
open is imho incorrect. with the open command, the app launch a external app to show this file, but this is a directory